### PR TITLE
[ACS-8597] use endpoint for bulk hold operation status

### PR DESCRIPTION
**Pull request by g-jaskowski**
**Please check if the PR fulfills these requirements**

> - [x] The commit message follows our [guidelines](https://github.com/Alfresco/alfresco-ng2-components/wiki/Commit-format)
> - [x] Tests for the changes have been added (for bug fixes / features)
> - [x] Docs have been added / updated (for bug fixes / features)

**What kind of change does this PR introduce?** (check one with "x")

> - [ ] Bugfix
> - [x] Feature
> - [ ] Code style update (formatting, local variables)
> - [ ] Refactoring (no functional changes, no api changes)
> - [ ] Build related changes
> - [ ] Documentation
> - [ ] Other... Please describe:

**What is the current behaviour?** (You can also link to an open issue here)
https://hyland.atlassian.net/browse/ACS-8597

**What is the new behaviour?**
Added api for checking bulk operation status and model for assign response entry 

**Does this PR introduce a breaking change?** (check one with "x")

> - [ ] Yes
> - [x] No

If this PR contains a breaking change, please describe the impact and migration path for existing applications: ...

**Other information**:

### CODE REVIEW

File: lib/js-api/src/api/gs-core-rest-api/docs/BulkAssignHoldResponseEntry.md [hunk 1]
```diff
@@ -0,0 +1,7 @@
+# BulkAssignHoldResponseEntry
+
+## Properties
+
+| Name      | Type                                                    | Default value | Description                                        |
+| --------- | ------------------------------------------------------- | ------------- | -------------------------------------------------- |
+| **Entry** | [**BulkAssignHoldResponse**](BulkAssignHoldResponse.md) | **null**      | Response received after bulk assign hold operation |
```
Comment (DaryaBalvanovich) [inline lib/js-api/src/api/gs-core-rest-api/docs/BulkAssignHoldResponseEntry.md:7]: could you please remove `**null**` for consistency with other files?

File: lib/js-api/src/api/gs-core-rest-api/model/holdBulkStatus.ts [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+import { HoldBulkOperation } from './holdBulkOperation';
+import { DateAlfresco } from '../../content-custom-api';
+
+export class HoldBulkStatus {
+    bulkStatusId: string;
+    errorsCount: number;
+    holdBulkOperation: HoldBulkOperation;
+    processedItems: number;
+    startTime: Date;
+    endTime?: Date;
```
Comment (DaryaBalvanovich) [inline lib/js-api/src/api/gs-core-rest-api/model/holdBulkStatus.ts:27]: could you pleas remove optional property to the end of the list?

**Review by DaryaBalvanovich — APPROVED** [review]

File: lib/js-api/src/api/gs-core-rest-api/docs/BulkAssignHoldResponseEntry.md (hunk 1 above)
Comment (g-jaskowski) [reply lib/js-api/src/api/gs-core-rest-api/docs/BulkAssignHoldResponseEntry.md]: removed, thanks!

File: lib/js-api/src/api/gs-core-rest-api/model/holdBulkStatus.ts (hunk 2 above)
Comment (g-jaskowski) [reply lib/js-api/src/api/gs-core-rest-api/model/holdBulkStatus.ts]: moved to the end in docs and model, thanks!

**Review by MichalKinas — APPROVED** [review]

**Pull request merged**